parser_library/src/parsing/grammar/macro_operand_rules.g4

```diff
@@ -53,77 +53,88 @@ mac_preproc[int* paren_count]
 		(
 			ORDSYMBOL (ORDSYMBOL|NUM)*
 			|
-			LPAR {++*$paren_count;}
+			LPAR
 			|
 			AMPERSAND
 		)
 		|
-		LPAR {++*$paren_count;}
+		LPAR
 		|
-		RPAR {--*$paren_count;}
+		RPAR
 		|
 		APOSTROPHE
 		(~(APOSTROPHE|ATTR|CONTINUATION))*
 		(APOSTROPHE|ATTR)
 		|
 		ATTR
 		(
-			{!is_previous_attribute_consuming(*$paren_count == 0, _input->LT(-2))}?
+			{is_attribute_consuming(_input->LT(-2))}?
+			AMPERSAND
+			(ORDSYMBOL (ORDSYMBOL|NUM)*|AMPERSAND)
+			|
+			{!is_attribute_consuming(_input->LT(-2)) || !can_attribute_consume(_input->LT(1))}?
```

Comment:
`I'&(&A)` also does not look string-like, which brings me back to 
```suggestion
			{!is_attribute_consuming(_input->LT(-2)) || (!can_attribute_consume(_input->LT(1)) && _input->LA(1) != AMPERSAND)}?
```

Reply:
I'll try to take another look in the morning but with the proposed change it parses remarks correctly but it doesn't work with e.g.

```
&INSTR   SETC   'J'
         MAC_STR2 I'&INSTR'
```

nor with

```
         MAC_STR2 I'&&'
```